_methods/card_sorting.markdown

```diff
@@ -1,16 +1,19 @@
+
 ---
 title: Card Sorting
 goal: Discover
+expert: Noah Manger, Andrew Maier
+
+what: <p>Card sorting provides users with a means of telling designers how they think about the information comprising our designs.</p>
 
-why: <p>To identify parts of the interaction that are hurdles. To generate new ideas for solutions that are grounded in the physical experience of the problem. Also, physical interactions generate empathy!</p>
+why: <p>Card sorting is a useful tool for helping designers create and structure content in a consistent, intuitive way.</p>
 
 how: |
-    <p>There are three levels of bodystorming, choose the level that’s right for the group and incorporates the right level of discomfort for a new group and new activity:</p>
-    <ul>
-      <li>Level 1: Go to the site you are studying, and do your work as normal. Don’t do any analysis, but be open to environmental cues and information that come from the observation.</li>
-      <li>Level 2: Use role play and props to start thinking through a new idea. Assign roles to people, making them either personas or objects, abstract or physical. It may be helpful for first-timers to assign the roles beforehand and start with a basic script.</li>
-      <li>Level 3:  Reflect on what happens, and why.</li>
-    </ul>
+    <p>There are two types of card sorting, open and closed. An open card sort is an exercise in which users are given a set of content and asked to group that content into whatever categories make sense. Users are then asked to label those categories. A closed card sort is the same thing, except users are given a predefined list of categories into which they should sort the content. Regardless of whether your card sort is open or closed, ask users to tell you about why they chose the categories they chose.</p>
+
+gov-context: 
+	<p>No PRA implications. The PRA explicitly exempts direct observation and non-standardized conversation (1320.3.H.3). It also specifically excludes tests of knowledge or aptitude (1320.3.H.7), which is essentially what a card sort tests (though in our case a poor result is our fault). That means: Sort away, no PRA clearance needed!</p>
+    
```

Comment:
Maybe check with Russ about whether we can use GSA's subscription to Optimal Sort? I think Noah used the free version but it would be great to figure out if we can access one licensed to the DigitalGov folks.